requirements.txt

```
@@ -7,7 +7,7 @@ matplotlib
 #qml
 # tqdm
 # Developers tools for compiling documentation
-sphinx>=2.1.2
+sphinx>=2.1.2,<=4.0.2
```

I don't think the doc would build with sphinx 2 😄 
```suggestion
sphinx==4.0.2
```

---

It actually does run with 2.1.2

---

ok, fair enough then =)